_inc/client/plans/single-product-backup.scss

```diff
@@ -42,6 +60,25 @@
 	color: #a7aaad;
 }
 
+.plan-radio-button__container {
+	display: inline-grid;
```

Comment:
Heads-up: we should make sure this works on IE11 as I'm not sure if autoprefixer takes care of CSS grid well enough.